features/option-list.feature

```diff
@@ -38,13 +38,24 @@ Feature: List WordPress options
     When I run `wp option add sample_test_field_one sample_test_field_value_one`
     And I run `wp option add sample_test_field_two sample_test_field_value_two`
     And I run `wp option list --search="sample_test_field_*" --format=csv`
+    And I run `wp option list --search="sample_test_field_*" --format=csv --orderby=option_name --order=asc`
+    And I run `wp option list --search="sample_test_field_*" --format=csv --orderby=option_value --order=asc`
```

Comment:
Why do you run these commands twice?

Reply:
@danielbachhuber all three commands have the same output. That's why I added like this.

Should I make it separate?

Reply:
You can do that like following.
http://docs.behat.org/en/v2.5/guides/1.gherkin.html

```
    When I run `wp option add sample_test_field_one sample_test_field_value_one`
    And I run `wp option add sample_test_field_two sample_test_field_value_two`
    And I run `wp option list --search="sample_test_field_*" --format=csv`
    Then STDOUT should be:
      """
      option_name,option_value
      sample_test_field_one,sample_test_field_value_one
      sample_test_field_two,sample_test_field_value_two
      """

    When I run `wp option add sample_test_field_one sample_test_field_value_one`
    And I run `wp option add sample_test_field_two sample_test_field_value_two`
    And I run `wp option list --search="sample_test_field_*" --format=csv --orderby=option_name --order=asc`
    Then STDOUT should be:
      """
      option_name,option_value
      sample_test_field_one,sample_test_field_value_one
      sample_test_field_two,sample_test_field_value_two
      """

    When I run `wp option add sample_test_field_one sample_test_field_value_one`
    And I run `wp option add sample_test_field_two sample_test_field_value_two`
    And I run `wp option list --search="sample_test_field_*" --format=csv --orderby=option_value --order=asc`
    Then STDOUT should be:
      """
      option_name,option_value
      sample_test_field_one,sample_test_field_value_one
      sample_test_field_two,sample_test_field_value_two
      """
```

Reply:
Thanks @miya0001